pkg/game/route.go

```diff
@@ -0,0 +1,16 @@
+package game
+
+import (
+	"github.com/KnoblauchPilze/user-service/pkg/rest"
+	"github.com/labstack/echo/v4"
+)
+
+func NewRoute(method string, path string, handler echo.HandlerFunc, actions ActionService) rest.Route {
+	wrapped := ActionWatcher(actions, handler)
+	return rest.NewRoute(method, false, path, wrapped)
```

Comment:
The new route definition: it builds on the existing `rest.Route` and automatically attaches a middleware to process the actions. It also by default only create unauthorized routes: this is because we anyway have some logic in [traefik](https://github.com/Knoblauchpilze/user-service/blob/master/deployments/config/stellar-dominion-service.yml#L10) to protect the endpoints for the `stellar-dominion-service` from public access.